### PR TITLE
[RM] Changes 8.13

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

**Comment by gares** [comment]
There are a couple of glitches in the rendered text, help is more than welcome

**Comment by jfehrle** [comment]
I can take a look.  You'd likeHelp by pushing additional commits here?

**Comment by gares** [comment]
sure, push as you like.

**Comment by gares** [comment]
I was also wondering if I should sort changes by type: Fix, then Change... right now it's all mixed

File: doc/sphinx/changes.rst [hunk 1]
```diff
@@ -8,6 +8,568 @@ Recent changes
...
+
+Tactics
+^^^^^^^
+
+- **Added:**
+  :tacn:`lia` is extended to deal with boolean operators e.g. `andb` or `Z.leb`.
+  (As `lia` gets more powerful, this may break proof scripts relying on `lia` failure.)
+  (`#11906 <https://github.com/coq/coq/pull/11906>`_,  by Frédéric Besson).
+- **Added:**
+  `apply in` supports several hypotheses
```
Comment (jfehrle) [inline doc/sphinx/changes.rst:285]:
```suggestion
  :tacn`apply … in` supports several hypotheses
```

File: doc/sphinx/changes.rst [hunk 2]
```diff
@@ -8,6 +8,568 @@ Recent changes
...
+  (`#12246 <https://github.com/coq/coq/pull/12246>`_,
+  by Hugo Herbelin; grants
+  `#9816 <https://github.com/coq/coq/pull/9816>`_).
+- **Removed:**
+  The deprecated and undocumented "prolog" tactic was removed
+  (`#12399 <https://github.com/coq/coq/pull/12399>`_,
+  by Pierre-Marie Pédrot).
+- **Removed:** Removed info tactic that was deprecated in 8.5.
```
Comment (jfehrle) [inline doc/sphinx/changes.rst:293]:
```suggestion
- **Removed:** Removed `info` tactic that was deprecated in 8.5.
```

File: doc/sphinx/changes.rst [hunk 3]
```diff
@@ -8,6 +8,568 @@ Recent changes
...
+  (`#12423 <https://github.com/coq/coq/pull/12423>`_, by Jim Fehrle).
+- **Added:**
+  Thhe :tacn:`zify` tactic can now be extended by redefining the `zify_pre_hook`
```
Comment (jfehrle) [inline doc/sphinx/changes.rst:296]:
```suggestion
  The :tacn:`zify` tactic can now be extended by redefining the `zify_pre_hook`
```

File: doc/sphinx/changes.rst [hunk 4]
```diff
@@ -8,6 +8,568 @@ Recent changes
...
+  (`#12648 <https://github.com/coq/coq/pull/12648>`_,  by Frédéric Besson).
+- **Changed:**
+  Giving an empty list of occurrences after :n:`in` in tactics is no
+  longer permitted. Omitting the :n:`in` gives the same behavior
+  (`#13237 <https://github.com/coq/coq/pull/13236>`_,
+  fixes `#13235 <https://github.com/coq/coq/issues/13235>`_,
+  by Hugo Herbelin).
+- **Fixed:**
+  Avoiding exposing an internal name of the form :n:`_tmp` when applying the
+  :n:`_` introduction pattern would break a dependency
```
Comment (jfehrle) [inline doc/sphinx/changes.rst:310]:
```suggestion
  :n:`_` introduction pattern which would break a dependency
```

File: doc/sphinx/changes.rst [hunk 5]
```diff
@@ -8,6 +8,568 @@ Recent changes
...
+  fixes `#13363 <https://github.com/coq/coq/issues/13363>`_,
+  by Hugo Herbelin).
+- **Deprecated:**
+  Undocumented :n:`eauto @nat_or_var @nat_or_var` syntax in favor of new :tacn:`bfs eauto`.
+  Also deprecated 2-integer syntax for :tacn:`debug eauto` and :tacn:`info_eauto`.
+  (Use :tacn:`bfs eauto` with the :flag:`Info Eauto` or :flag:`Debug Eauto` flags instead.)
+  (`#13381 <https://github.com/coq/coq/pull/13381>`_,
+  by Jim Fehrle).
+- **Removed:**
+  :n:`at @occs_nums` clauses in tactics such as tacn:`unfold`
```
Comment (jfehrle) [inline doc/sphinx/changes.rst:327]:
```suggestion
  :n:`at @occs_nums` clauses in tactics such as :tacn:`unfold`
```

File: doc/sphinx/changes.rst [hunk 6]
```diff
@@ -8,6 +8,568 @@ Recent changes
...
+  values.  These include {e}constructor, do, timeout,
+  9 {e}auto tactics and psatz*.
+  (`#13417 <https://github.com/coq/coq/pull/13417>`_,
+  by Jim Fehrle).
+
+Tactic language
+^^^^^^^^^^^^^^^
+
+- **Fixed:**
+  printing of the quotation qualifiers when printing :g:`Ltac` functions
```
Comment (jfehrle) [inline doc/sphinx/changes.rst:345]:
```suggestion
  Printing of the quotation qualifiers when printing :g:`Ltac` functions
```

File: doc/sphinx/changes.rst [hunk 7]
```diff
@@ -8,6 +8,568 @@ Recent changes
...
+- **Removed:**
+  In the :cmd:`Extraction Language` command, remove `Ocaml` as a valid value.
+  Use `OCaml` instead.  This was deprecated in Coq 8.8, `#6261 <https://github.com/coq/coq/pull/6261>`_
+  (`#13016 <https://github.com/coq/coq/pull/13016>`_, by Jim Fehrle).
+- **Changed:**
+  When compiled with OCaml >= 4.10.0, Coq will use the new best-fit GC
+  policy, which should provide some performance benefits. Coq's policy
+  is optimized for speed, but could increase memory consumption in
+  some cases. You are welcome to tune it using the ``OCAMLRUNPARAM``
+  variable and report back setting so we could optimize more.
```
Comment (jfehrle) [inline doc/sphinx/changes.rst:388]:
```suggestion
  variable and report back on good settings so we can improve the defaults.
```

File: doc/sphinx/changes.rst [hunk 8]
```diff
@@ -8,6 +8,568 @@ Recent changes
...
+  (`#13345 <https://github.com/coq/coq/pull/13345>`_,
+  fixes `#13344 <https://github.com/coq/coq/issues/13344>`_,
+  by Hugo Herbelin).
+- **Changed:**
+  Option -native-compiler of the configure script now impacts the
+  default value of the option -native-compiler of coqc. The
+  -native-compiler option of the configure script is added an ondemand
+  value, which becomes the default, thus preserving the previous default
+  behavior.
+  The stdlib is still precompiled when configuring with -native-compiler
```
Comment (jfehrle) [inline doc/sphinx/changes.rst:427]:
```suggestion
- **Changed:**
  Option `-native-compiler` of the configure script now impacts the
  default value of the `-native-compiler` option of coqc. The
  `-native-compiler` option of the configure script is added as an on demand
  value, which becomes the default, thus preserving the previous default
  behavior.
  The stdlib is still precompiled when configuring with `-native-compiler`
```

File: doc/sphinx/changes.rst [hunk 9]
```diff
@@ -8,6 +8,568 @@ Recent changes
...
+  :attr:`global` otherwise, but is scheduled to change in a future release. For the
+  time being, adding hints outside of sections without specifying an explicit
+  locality is therefore triggering a deprecation warning. It is recommended to
+  use :attr:`export` whenever possible
+  (`#13384 <https://github.com/coq/coq/pull/13384>`_,
+  by Pierre-Marie Pédrot).
+- **Changed:**
+  The :attr:`export` locality can now be used for all Hint commands,
+  including Hint Cut, Hint Mode, Hint Transparent / Opaque and
+  Remove Hints
```
Comment (jfehrle) [inline doc/sphinx/changes.rst:444]:
```suggestion
  including :tacn:`Hint Cut`, :tacn:`Hint Mode`, :tacn:`Hint Transparent` / `Opaque` and
  :tacn:`Remove Hints`
```

File: doc/sphinx/changes.rst [hunk 10]
```diff
@@ -8,6 +8,568 @@ Recent changes
...
+  (`#13388 <https://github.com/coq/coq/pull/13388>`_,
+  by Pierre-Marie Pédrot).
+
+Tools
+^^^^^
+
+- **Changed:**
+  Adding the possibility in coq_makefile to directly set the installation folders,
```
Comment (jfehrle) [inline doc/sphinx/changes.rst:452]:
```suggestion
  Added the ability for coq_makefile to directly set the installation folders,
```

File: doc/sphinx/changes.rst [hunk 11]
```diff
@@ -8,6 +8,568 @@ Recent changes
...
+  by Li-yao Xia).
+
+Infrastructure and dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  Coq's core system now uses the `zarith <https://github.com/ocaml/Zarith>`_
+  library, based on GNU's gmp instead of ``num`` which is
+  deprecated upstream. The custom ``bigint`` module is
+  not longer provided; note that the ``micromega`` still uses
```
Comment (jfehrle) [inline doc/sphinx/changes.rst:548]:
```suggestion
  not longer provided; note that ``micromega`` still uses
```

**Review by jfehrle — COMMENTED** [review]
A few notable glitches and some wording tweaks.

Hope this helps.

**Comment by gares** [comment]
Thanks

**Comment by mattam82** [comment]
I'm going to cherry-pick from here to #13527

**Pull request closed without merging**